tensorflow/core/debug/debug_grpc_io_utils_test.cc

```diff
@@ -25,6 +25,9 @@ limitations under the License.
 #include "tensorflow/core/platform/env.h"
 #include "tensorflow/core/util/event.pb.h"
 
+// TODO(cais): Support grpc:// debug URLs in open source once Python grpc
+//   genrule becomes available. See b/23796275.
+#if defined(PLATFORM_GOOGLE)
```

Comment:
The problem is that there doesn't seem to be a macro that I can use for cmake builds specifically. So if I just excluded this test from tf_tests.cmake, the test would still be run in Linux cmake builds and fail because of the PLATFORM_GOOGLE macro I applied in debug_io_utils.cc. Do you know of a macro defined specifically for cmake builds? If not, I think the approach here is the cleanest. This test used to run and pass in open-source bazel tests (e.g., on Linux and Mac), but since the server side of grpc:// debug URLs is not defined in open source anyway, open-source users aren't able to use the feature yet anyway. I don't think it's going it's going to cause any problems to exclude it for now.